solvers/src/macro_solver/solver.rs

```diff
@@ -116,7 +116,7 @@ impl<'a> MacroSolver<'a> {
             }
 
             popped += 1;
-            if popped % (1 << 14) == 0 {
+            if popped % (1 << 6) == 0 {
```

Comment:
I feel like this update rate is a bit much. How many updates per second is this approximately? I haven't measured the cost of the callback, but I'm sure it's not something we want to be invoking potentially hundreds of thousands of times per solve.

Reply:
I just tested `1 << 6` and even on the web version I think the update rate is just too much. I think `1 << 10` is a good compromise (still updates very quickly but you only get the update overhead once every 1024 nodes instead of once every 64 nodes. What do you think?

Reply:
[Screencast from 2024-11-29 22-13-05.webm](https://github.com/user-attachments/assets/7dfa5b13-bf5c-4c5e-a2a6-30aba40f34e7)